frontend/package.json

```diff
@@ -13,40 +13,38 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/common": "^2.2.4",
-    "@angular/compiler": "^2.2.4",
-    "@angular/compiler-cli": "^2.2.4",
-    "@angular/core": "^2.2.4",
-    "@angular/forms": "^2.2.4",
-    "@angular/http": "^2.2.4",
-    "@angular/platform-browser": "^2.2.4",
-    "@angular/platform-browser-dynamic": "~2.2.4",
-    "@angular/platform-server": "~2.2.4",
-    "@angular/router": "^3.2.4",
-    "@ngrx/core": "^1.2.0",
-    "@ngrx/effects": "^2.0.0",
-    "@ngrx/store": "^2.2.1",
-    "@ngrx/store-devtools": "^3.2.2",
-    "@ngx-translate/core": "^7.2.2",
-    "@ngx-translate/http-loader": "<1.0.0",
-    "@types/lodash": "^4.14.50",
-    "angular2-jwt": "^0.1.25",
+    "@angular/animations": "^5.2.11",
```

Comment:
why not latest version?

Reply:
My strategy for this was to gradually migrate between versions in order to have smaller commits and fallback versions in case something fails or requires more changes than expected. 
My plan is to eventually migrate it to latest available, as the title of the PR.
Sorry for the misleading title. I will update it.